forestserviceprototype/specialuseform/templates/specialuseform/home.html

```diff
@@ -0,0 +1,14 @@
+{% extends "specialuseform/base.html" %}
+
+{% block content %}
+<h1>Welcome to the Forest Service Online Permits System</h1>
+
+<p>Are you an commerical outfitter?</p>
+<small>Coming Soon</small>
+<div class="button_wrapper">
+   <button class="usa-button-disabled" disabled>Yes</button>
+ </div>
+
+<p>Are you looking to run an event in the forest?</p>
```

Comment:
I think we probably want this to be something like: 

**Are you organizing a _non-commercial_ activity that involves a group of 75 or more people?** (either as participants or spectators). Noncommercial is any use or activity where an entry or participation fee is not charged, and the primary purpose is not the sale of a goods or service. For example: weddings, church services, endurance rides, regattas, camping trips, hikes, music festivals, rallies, graduations, and races.

From: http://www.fs.fed.us/specialuses/special_non_com_uses.shtml#sp-noncom-a